templates/flavors/dual-stack/kustomization.yaml

```diff
@@ -0,0 +1,42 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+  - ../default
+
+patches:
+  - target:
+      group: cluster.x-k8s.io
+      version: v1beta1
+      kind: Cluster
+    patch: |-
+      apiVersion: cluster.x-k8s.io/v1beta1
+      kind: Cluster
+      metadata:
+        name: ${CLUSTER_NAME}
+        labels:
+          ipv6: true
```

Comment:
Should this be `ipv6: "true"` as in addons/cilium/cilium-ipv6.yaml or without quotes here? Didn't install it, hence checking.

Reply:
Yeah I had that at one point on a different branch, updated 👍